README.md

```diff
@@ -27,3 +27,44 @@ npm install libyear-npm --save-dev
      validator      6.3.0      2017-02-24      7.0.0      2017-02-24     0.0
 System is 1.8 libyears behind
 ```
+
+### Programmatic usage
+
+It is possible to use libyear-npm from your applications:
+
+```
+var libyear = require('libyear-npm');
```

Comment:
Please make sure the example works. 
I tried quickly (using `lib/index.js`) but I get this error: 

![image](https://user-images.githubusercontent.com/1448836/71563126-76fcf780-2a58-11ea-8c0c-6baa8f88d77e.png)